test/e2e/Makefile

```diff
@@ -16,17 +14,22 @@ docker-debug:
 		--tag cometbft/e2e-node:local-version \
 		-f docker/Dockerfile.debug ../..
 
+docker-fast:
+	@echo "Building fast-prototyping E2E Docker image"
+	@docker build \
+		--tag cometbft/e2e-node:local-version \
+		-f docker/Dockerfile.fast .
+
 # We need to build support for database backends into the app in
 # order to build a binary with a CometBFT node in it (for built-in
 # ABCI testing).
 node:
-	go build -race $(BUILD_FLAGS) -tags '$(BUILD_TAGS)' -o build/node ./node
-
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o build/node ./node
 
 generator:
-	go build -o build/generator ./generator
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o build/generator ./generator
```

Comment:
This action applies also for the existing Makefile roles. Are you sure they work the same with these flags?